test/new_relic/agent/transaction/external_request_segment_test.rb

```diff
@@ -868,7 +868,7 @@ def test_non_sampled_segment_does_not_record_span_event
         assert_empty last_span_events
       end
 
-      def test_non_sampled_segment_does_not_record_span_event
+      def test_ignored_segment_does_not_record_span_event
```

Comment:
this change is unrelated, it's just because I noticed a warning that this test method was being defined twice, so I wanted to get that fixed as well.